### ISSUE TITLE
Cannot add node "1" because a node with that id is already

### ISSUE THREAD

**Issue by DongHeonRyu** [jhen0409/react-native-debugger#708]
React Native Debugger app version: download it from howbrew 
React Native version: 0.68.7 
Platform:android 
Is real device of platform: no 
Operating System: mac os

![Screen Shot 2022-07-02 at 10 41 55 PM](https://user-images.githubusercontent.com/60118440/177026622-4813c7f1-4890-4120-b553-f3386dcad5e3.png)

I did  everything here https://github.com/jhen0409/react-native-debugger/issues/668

but whenever I did

```

on Mac OS (M1) the following fixed it for me:

yarn global add react-devtools react-devtools-core
and, in the app's package.json:

"resolutions": {
    "react-devtools-core": "4.14.0"
  },
```

The metro terminal show 

Error: Unable to resolve module `./debugger-ui/debuggerWorker.d9da4ed7` from ``:
None of these files exist:
  * debugger-ui/debuggerWorker.d9da4ed7(.native|.native.js|.js|.native.json|.json|.native.ts|.ts|.native.tsx|.tsx)

and the RNdebugger application is keep blinking

**Comment by ajp8164** [comment]
Seeing same error and also added the deps etc as described in #668.

RNB 0.12.1

from package.json
```
  "dependencies": {
...
    "react": "18.0.0",
    "react-devtools": "4.14.0",
    "react-devtools-core": "4.14.0",
    "react-native": "0.69.1"
},
  "resolutions": {
    "react-devtools-core": "4.14.0"
  },
```

**Comment by mohan1361** [comment]
Seeing same error so i fixed it for RND 0.12.1
in package.json just add  
` "resolutions": {
        "react-devtools-core": "~4.14.0"
    },`
    
    and remove node_modules and package.json
`     rm package-lock.json 
     rm  node_modules`
     and check if you not installed yarn then install globally
     `npm install --global yarn`
     and check yarn --version if it installed
     and simply run 
     yarn install
     then in your yarn.lock you see this file and this generally override dev-tools
     `react-devtools-core@^4.23.0, react-devtools-core@~4.14.0:` and you can confirm it in yarn.lock.
     
![image](https://user-images.githubusercontent.com/68941763/182316598-226fdbb1-49f0-4fd5-946f-aef51c206c74.png)

**Comment by jhen0409** [comment]
It fixed on `v0.13.0`. It also required bump `react-devtools-core` to v4.25 on project.
